tests/testthat/test_geo.R

```diff
@@ -14,4 +14,48 @@ test_that("ward_enr_aggs correctly labels newark 2018-19 enrollment data", {
 test_that("ward_enr_aggs correctly aggregates newark enrollment data by ward", {
   enr_2019_wards <- ward_enr_aggs(enr_2019)
   testthat::expect_is(enr_2019_wards, 'data.frame')
+})
+
+
+
+# PARCC
+parcc_ela_8_2019 <- fetch_parcc(end_year = 2019, grade_or_subj = 8, 
+                          subj = 'ela', tidy = T)
+
+test_that("ward_parcc_aggs correctly aggregates newark parcc data by ward" , {
+   parcc_ela_8_2019 %>%
+      ward_parcc_aggs() %>%
+      testthat::expect_is('data.frame')
+})
+
+test_that("ground truth values for parcc ward aggregations", {
+   parcc_ela8_19_ward <- parcc_ela_8_2019 %>%
+      ward_parcc_aggs()
+   
+   expect_equal(parcc_ela8_19_ward %>%
+                   filter(subgroup == "special_education",
+                          district_name == "Newark City CENTRAL") %>%
+                   pull(number_of_valid_scale_scores),
+                54)
+   
+   expect_equal(parcc_ela8_19_ward %>%
```

Comment:
I got rid of this test - one of the few fields that didn't make it with the port to the `parcc_calcs` aggregation code.  I think it's for the best because `n()` will give you rows in the group, but that's not necessarily the equivalent of number of schools.  To get number of schools I think we need to count the unique schoolids (possibly excluding 'district' aggregation rows)